sources/advanced/builtins/index.ts

```diff
@@ -1,3 +1,3 @@
-export * from './definitions'
-export * from './help'
-export * from './version'
\ No newline at end of file
+export * from './definitions';
+export * from './help';
+export * from './version';
```

Comment:
Does the eslint config not enforce a newline at the end of files? :thinking:

Reply:
Looks like it doesn't:
![image](https://user-images.githubusercontent.com/32596136/103319289-f2adcd80-4a39-11eb-94a2-509c443a9727.png)

Reply:
https://github.com/yarnpkg/berry/pull/2287